pkg/cmd/cli/cmd/exporter.go

```diff
@@ -77,19 +75,10 @@ func (e *defaultExporter) Export(obj runtime.Object, exact bool) error {
 		pod.Strategy.PrepareForCreate(obj)
 	case *kapi.PodTemplate:
 	case *kapi.Service:
-		// TODO: service does not yet have a strategy
-		t.Status = kapi.ServiceStatus{}
-		if exact {
-			return nil
-		}
+		// moved in the server
 		if t.Spec.ClusterIP != kapi.ClusterIPNone {
 			t.Spec.ClusterIP = ""
```

Comment:
@smarterclayton upstream server-side they are adding "<unknown>" instead of leaving this empty: https://github.com/kubernetes/kubernetes/blob/35ea9b87b82992c044fffb27afd85ddfbd173666/pkg/registry/service/strategy.go#L93
That's the only difference between `oc get --export` and `oc export` right now. I am preserving this for `oc export` but I am not sure why one should deviate.